backend/substrapp/tests/views/tests_views_compositealgo.py

```diff
@@ -138,6 +138,24 @@ def test_composite_algo_list_filter_objective_fail(self):
 
             self.assertIn('Malformed search filters', r['message'])
 
+    def test_composite_algo_list_filter_model(self):
+        url = reverse('substrapp:composite_algo-list')
+        done_model = [
```

Comment:
shouldn't we use use a `filter` here?
```
done_models = filter(lambda m: 'compositeTraintuple' in m and m['compositeTraintuple']['status'] == 'done' > 18, model)
done_model = done_models[0]
```

Reply:
We never use filters in our codebase but rather list comprehensions such as this one. I stuck with the current pattern to be consistent (this new test is adapted from existing ones). And from what I'm reading, the functions map/filter/reduce are at the center of quite a hot topic in the python community :)

Reply:
`filter` is better in a pythonic way : explicit before implicit.
Not using it in our current codebase is not a sign of good health.

Reply:
And yet Guido Van Rossum wanted it out of the language. All I'm saying is that consistency trumps other arguments here and that this small PR isn't the best place to have this discussion.

Reply:
Both syntaxes are valid and could be used. 
Personally,  I'm using more often the list comprehension as I find it more readable and easier to maintain.